process_material/models.py

```diff
@@ -0,0 +1,53 @@
+from django.db import models
+from django.contrib.auth.models import (AbstractUser, AbstractBaseUser,
+                                        PermissionsMixin)
+
+class User(AbstractUser):
+
+    archivist_group = [u"rac_archivists"]
+    donor_group = [u"rac_donors"]
+    researcher_group = [u"rac_researchers"]
+
+    email = models.EmailField(
+        verbose_name=_('email address'), max_length=255, unique=True, blank=False)
+    first_name = models.CharField(_('first name'), max_length=150, blank=False)
```

Comment:
I've overridden attributes on existing fields like the below. I'm not 100% sure what the best practice is though.

```suggestion
    AbstractUser._meta.get_field("first_name").blank = False
```